tensorflow/compiler/xla/service/gpu/tests/reduction_vectorization_test.cc

```diff
@@ -46,6 +46,10 @@ class ReductionVectorizationNoOptTest : public GpuCodegenTest {
 };
 
 TEST_F(ReductionVectorizationNoOptTest, MultiOutputStore) {
+  if (GetCudaComputeCapability().IsAtLeast(se::CudaComputeCapability::PASCAL_)) {
```

Comment:
Forgot "!" ?

Reply:
Right. I also fixed the commit author to fix the CLA.